snippets/legv8.json

```diff
@@ -0,0 +1,185 @@
+{
+  "set register to": {
+    "prefix": "sreg",
+    "body": ["ADD ${1:dest reg}, XZR, $2", "$3"]
+  },
```

Comment:
Add description for all of them. Also try to make it easy to realize what the command does.

So if the code inserted is something to the effect of `ADDI reg, reg, #1`, show how to think of it in basic math terms, like `reg = reg + 1`

Reply:
Good comment! It`s implemented